cmd/swctl/main.go

```diff
@@ -102,6 +103,7 @@ services, service instances, etc.`
 		alarm.Command,
 		layer.Command,
 		process.Command,
+		profiling.Command,
```

Comment:
You had `profile` command, now you have `profiling` command, this is really confusing....

Reply:
Yes. but for now, i cannot find out a good way to split them. How about update the old one to "trace-profiling", and the new one is "ebpf-profiling". They are two different profiling way, but it will break the old command. Do you have any good idea?

Reply:
If I'm a user I'd expect the commands look like 

```shell
swctl profiling trace
swctl profiling cpu
swctl profiling memory
```

I don't care what technology you use, eBPF or others.

That's just my 2 cents. Here my concern is all about the codes not features, but the 2 commands are too similar that I don't think I can ignore them

Reply:
update to two sub-commands: `profiling trace` and `profiling ebpf`.